prisma/schema.prisma

```diff
@@ -85,3 +87,13 @@ model ViewSetting {
   @@index([userId, workspaceId], name: "IX_ViewSettings_userId_workspaceId")
   @@map("ViewSettings")
 }
+
+model Attachment {
+  id          String   @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  task        Task     @relation(fields :[taskId], references : [id])
+  taskId      String   @db.Uuid
+  url         String   
+  createdById String   @db.Uuid
```

Comment:
Please add a `workspaceId` field to this table to check whether the user's workspace owns the attachments or not. If we don't filter this way, users can easily read, delete, etc the attachments of another workspace,  which is a security concern.

So basically on any CRUD operation you have to first check if the `workspaceId` of the attachment matches the workspaceId extracted from token payload or not